state/src/storage/tx/mod.rs

```diff
@@ -107,6 +107,7 @@ impl<'db> StateTxn<'db, RW> {
             MPC_PREPROCESSING_TABLE,
             NODE_METADATA_TABLE,
             RAFT_METADATA_TABLE,
+            RAFT_LOGS_TABLE,
```

Comment:
do we need to increment `NUM_TABLES`?

Reply:
no, this was already created elsewhere i added it here for automatic creation